packages/permission-controller/src/PermissionController.ts

```diff
@@ -1573,8 +1581,26 @@ export class PermissionController<
       >
     >;
 
+    const populatedRequest =
```

Comment:
With the current permission tree approach if we _change a permission group after its creation_ a user might be able to get in a weird state where they don't have all the permissions they expect

Reply:
True, hmm...maybe that's something that should be taken care of in the migration?

Reply:
We need to be explicit with this at least, whether we expect migrations to be written when permission groups change or whether permission groups are dynamic and will automatically change when we update specifications

Reply:
I'll mention it in the docs, maybe even leave a code snippet for a migration.